packages/cli/src/commands/migrate/index.ts

```diff
@@ -77,8 +76,8 @@ async function migrate(options: MigrateCommandOptions): Promise<void> {
   // 1. Not --dryRun
   // 2. Not --regen
   // 3. First time run (check if any report exists)
-  if (!options.dryRun && !options.regen && !reportExisted(options.basePath, options.outputPath)) {
-    const hasUncommittedFiles = await gitIsRepoDirty(options.basePath);
+  if (!options.dryRun && !options.regen && !reportExisted(basePath, options.outputPath)) {
```

Comment:
Why don't keep basePath as part of *Options types with default ` = process.cwd()`?

You can use [.hideHelp()](https://www.npmjs.com/package/commander#user-content-more-configuration), or just don't expose this to CLI at all.

Reply:
I was hiding it exactly as you suggested here at first. But then thought if we just hide it and don't use it as an option, we could just remove it and take it out of `options`.

It makes more sense to me doing this way, since we agreed to make it a hard requirement to run the CLI from project root so that `basePath` won't be a variable in any case. However, if at some point we need to pass the `basePath` as option again, then for sure we need to keep it within `options`.